### ISSUE TITLE
File locations different. Remote software not detected by Reason 11

### ISSUE THREAD

**Issue by douglasgough** [spoida/reason-remote-komplete-kontrol#2]
Hi. Thanks for your work on this. Your instructions are to install these files inside c:/ProgramData/... but that folder structure doesn't exist on my Windows 10 computer. Instead, everything seems to be in C:\Program Files\Propellerhead\Reason 11\Remote. Within that directory I find folders for BuiltinCodecs, BuiltinMaps, DefaultCodecs, and DefaultMaps. Within DefaultCodecs there is a Lua Codecs/Native Instruments folder, and within DefaultMaps there is NativeInstruments folder. I have copied the reason-remote-komplete-control files into these folders but when I try to enable them in Reason there is no entry for Komplete Kontrol. I only see the Audio Kontrol 1 that came with Reason.

**Comment by douglasgough** [comment]
Reason gives this error report:
Remote Codec Errors: Failed to run .luacodec file: Komplete Kontrol MCU.luacodec
Komplete Kontrol MCU.luacodec, LUA Error: [string "Komplete Kontrol MCU.luacodec"]:7: unexpected symbol near '<'
Failed to run .luacodec file: Komplete Kontrol S61 MK2.luacodec
Komplete Kontrol S61 MK2.luacodec, LUA Error: [string "Komplete Kontrol S61 MK2.luacodec"]:7: unexpected symbol near '<'
Line  7: Missing: Propellerhead Remote Mapping File.
Failed to read map file: Komplete Kontrol MCU.remotemap.
Line  7: Missing: Propellerhead Remote Mapping File.
Failed to read map file: Komplete Kontrol S61 MK2.remotemap.

**Comment by douglasgough** [comment]
I just learned that ProgramData is a hidden folder. I unhid it and created the folder structure from the instructions, then copied the files in to the right places. I still get an error:

Remote Codec Errors: Failed to run .luacodec file: Komplete Kontrol S61 MK2.luacodec
Komplete Kontrol S61 MK2.luacodec, LUA Error: [string "Komplete Kontrol S61 MK2.luacodec"]:7: unexpected symbol near '<'
Line  7: Missing: Propellerhead Remote Mapping File.
Failed to read map file: Komplete Kontrol S61 MK2.remotemap.

**Comment by spoida** [comment]
Hi @douglasgough - sorry for the delay (I think I've somehow messed up my github email notifications and I'm no longer receiving messages when people create issues).

You are correct, ProgramData is a hidden folder. If you are copying these files manually then you'll need make it visible in Windows Explorer first. Sounds like you've go that part sorted 👍 

For the second part, it looks like the file contents have become corrupted somehow? Whenever you see things like "unexpected symbol" it means the file contents are not being recognised by Reason.

So, how did you create the files and copy the contents? If you are copying the contents from the Github UI (user interface) there's a chance the website has put some unsupported characters in the text of the file. To play it safe, it's better to download all of the files locally and just copy them verbatim. You can download everything using this link https://github.com/spoida/reason-remote-komplete-kontrol/archive/refs/heads/master.zip

Then unzip that file and copy things to the correct folders.  Please have a go at that and then let me know.  I have this working on Windows 10 with Reason 11 so we'll definitely be able to get you sorted - just need to figure out what's going wrong 😃

**Comment by douglasgough** [comment]
I should have downloaded the whole package as you suggest. What I did was
open each file in the Github editor and then copy and paste to an
identically named file on my system. I think I must have accidentally
selected the entire web page instead of just the content of the editor when
I copied. I've since replace my corrupted files with the proper files and
placed them in the ProgramData folder. Everything is working now. Thanks
for your help.

Thanks

Doug Gough
